test/base/tojs.js

```diff
@@ -298,9 +298,9 @@ test("verify #566 solution", () => {
     const b = mobx.observable({ x: 3 })
     const c = mobx.observable({ a: a, b: b })
 
-    expect(mobx.toJS(c).a === a).toBeTruthy() // true
-    expect(mobx.toJS(c).b !== b).toBeTruthy() // false, cloned
-    expect(mobx.toJS(c).b.x === b.x).toBeTruthy() // true, both 3
+    expect(mobx.toJS(c).a).toBe(a)
+    expect(mobx.toJS(c).b).toEqual(mobx.toJS(c.b))
```

Comment:
for completeness sake, restore the previous:

`expect(mobx.toJS(c).b).not.toEqual(b)`

Reply:
Reverted